idea/idea-core/src/org/jetbrains/kotlin/idea/core/util/PsiLinesUtils.kt

```diff
@@ -47,18 +49,26 @@ fun PsiElement.getLineCount(): Int {
     if (doc != null) {
         val spaceRange = textRange ?: TextRange.EMPTY_RANGE
 
-        if (spaceRange.endOffset <= doc.textLength) {
+        if (spaceRange.endOffset <= doc.textLength && spaceRange.startOffset < spaceRange.endOffset) {
             val startLine = doc.getLineNumber(spaceRange.startOffset)
             val endLine = doc.getLineNumber(spaceRange.endOffset)
 
-            return endLine - startLine
+            return endLine - startLine + 1
         }
     }
 
     return (text ?: "").count { it == '\n' } + 1
```

Comment:
Redundant `:?`

Reply:
It would be correct to use `com.intellij.openapi.util.text.StringUtil#getLineBreakCount` insted of `count`

Reply:
> Redundant :?

Not sure. It's `String!`. So for safety there's `?:`